operations/helm/charts/mimir-distributed/templates/graphite-proxy/graphite-web/graphite-web-dep.yaml

```diff
@@ -33,11 +33,7 @@ spec:
           imagePullPolicy: {{ .Values.graphite.web.image.pullPolicy }}
           env:
             - name: GRAPHITE_CLUSTER_SERVERS
-              {{- if .Values.gateway.useDefaultProxyURLs }}
```

Comment:
This just didn't make sense. There is no service under 
```
{{ template "mimir.fullname" . }}.{{ .Release.Namespace }}.svc:{{ include "mimir.serverHttpListenPort" . }}/graphite
```

I must've missed this during the graphite reviews